docs/source/tutorials_source/package/tutorial_active_learning.py

```diff
@@ -0,0 +1,185 @@
+
+
+
+
+# %%
+# Imports
+# -------
+#
+# Import the Python frameworks we need for this tutorial.
+
+import csv
+from typing import List, Dict, Tuple
+import numpy as np
+from sklearn.neighbors import KNeighborsClassifier
+
+from lightly.active_learning.agents.agent import ActiveLearningAgent
+from lightly.active_learning.config.sampler_config import SamplerConfig
+from lightly.active_learning.scorers.classification import ScorerClassification
+from lightly.api.api_workflow_client import ApiWorkflowClient
+from lightly.openapi_generated.swagger_client import SamplingMethod
+
+# %%
+# Definition of parameters
+path_to_embeddings_csv = "path/to/clothing-dataset-small/train/.../embeddings.csv"
+YOUR_TOKEN = "yourToken"  # your token of the web platform
+YOUR_DATASET_ID = "yourDatasetId"  # the id of your dataset on the web platform
+
+
+# %%
+# Definition of a dataset for the classifier based on the embeddings.csv
+# The kNN-classifier needs the embeddings as features for its classification. Thus we define a class to create
+# such a dataset out of the embeddings.csv. It also allows to choose only a subset of all samples
+# dependent on the filenames given.
+
+class CSVEmbeddingDataset:
+    def __init__(self, path_to_embeddings_csv: str):
+        with open(path_to_embeddings_csv, 'r') as f:
+            data = csv.reader(f)
+
+            rows = list(data)
+            header_row = rows[0]
+            rows_without_header = rows[1:]
+
+            index_filenames = header_row.index('filenames')
+            filenames = [row[index_filenames] for row in rows_without_header]
+
+            index_labels = header_row.index('labels')
+            labels = [row[index_labels] for row in rows_without_header]
+
+            embeddings = rows_without_header
+            indexes_to_delete = sorted([index_filenames, index_labels], reverse=True)
+            for embedding_row in embeddings:
+                for index_to_delete in indexes_to_delete:
+                    del embedding_row[index_to_delete]
+
+        # create the dataset as a dictionary mapping from the filename to a tuple of the embedding and the label
+        self.dataset: Dict[str, Tuple[np.ndarray, int]] = \
+            dict([(filename, (np.array(embedding_row, dtype=float), int(label)))
+                  for filename, embedding_row, label in zip(filenames, embeddings, labels)])
+
+    def get_features(self, filenames: List[str]) -> np.ndarray:
+        features_array = np.array([self.dataset[filename][0] for filename in filenames])
+        return features_array
+
+    def get_labels(self, filenames: List[str]) -> np.ndarray:
+        labels = np.array([self.dataset[filename][1] for filename in filenames])
+        return labels
+
+
+# %%
+# Upload the embeddings to the lightly web platform
+api_workflow_client = ApiWorkflowClient(token=YOUR_TOKEN, dataset_id=YOUR_DATASET_ID)
+api_workflow_client.upload_embeddings(name="embedding-1", path_to_embeddings_csv=path_to_embeddings_csv)
```

Comment:
This would still be replaced by the CLI now right? 

FIY because maybe you didn't know. You can embed, and upload in a single command like this:
```bash
lightly-magic input_dir=path/to/input/dir trainer.max_epochs=0 dataset_id=XYZ token=123
```

Reply:
No, it cannot, as the upload_embeddings sets the internal api_workflow_client.embedding_id.

Reply:
Hm ok that's not nice...

There's an endpoint at ` /v1/datasets/{datasetId}/embeddings` to get all embeddings by dataset id. Can we use this one instead and have the user pass the name of the embedding? 

@IgorSusmelj thoughts?

Reply:
I would make the following:
- user can also not provide an embedding_id --> we just take the latest one in the db

Cases, where I have multiple embeddings and want to take a specific one, are rare for now.

Reply:
So, just that I understand correctly, the user cannot even pass an embedding name? The program just picks one?

Reply:
The user could use 
```python
api_workflow_client.upload_embeddings(name="existing-embedding-name", path_to_embedding_csv="") 
```
to set the `api_workflow_client.embedding_id` for an embedding specified by the name.
I would propose  to have a separate method for that, but in another issue.